.github/workflows/unit-test.yml

```diff
@@ -15,14 +15,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, "3.10", 3.11]
```

Comment:
What was the reason for removing Python 3.7?

Reply:
https://devguide.python.org/versions

Supporting 5 simultaneous versions of Python seems like a reasonable workload.  It is best not to encourage the continued use of software which not longer receives security updates.